src/main/java/com/sayone/ebazzar/entity/AddressEntity.java

```diff
@@ -3,7 +3,7 @@
 
 import javax.persistence.*;
 
-@Entity(name="addressEntity")
+@Entity(name="address")
 public class AddressEntity {
```

Comment:
All tables should have created/modified timestamp. Create a base table and use it.